client/dashboard/task-list/tasks/shipping/index.js

```diff
@@ -0,0 +1,212 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import apiFetch from '@wordpress/api-fetch';
+import { Component } from '@wordpress/element';
+import { compose } from '@wordpress/compose';
+import { filter } from 'lodash';
+import { withDispatch } from '@wordpress/data';
+
+/**
+ * WooCommerce dependencies
+ */
+import { Card, Stepper } from '@woocommerce/components';
+import { getHistory, getNewPath } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import StoreLocation from './location';
+import ShippingRates from './rates';
+import withSelect from 'wc-api/with-select';
+
+class Shipping extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.initialState = {
+			isPending: false,
+			step: 'store_location',
+			shippingZones: [],
+		};
+
+		this.state = this.initialState;
+
+		this.completeStep = this.completeStep.bind( this );
+	}
+
+	componentDidMount() {
+		this.reset();
+	}
+
+	reset() {
+		this.setState( this.initialState );
+	}
+
+	async fetchShippingZones() {
+		this.setState( { isPending: true } );
+		const { countryCode, countryName } = this.props;
+		const shippingZones = [];
+		const zones = await apiFetch( { path: '/wc/v3/shipping/zones' } );
```

Comment:
I think this is fine for now, but we might want to add a `@todo` here for potentially moving this work over to selectors and mutators -- especially if code starts getting lifted from here for a settings screen overhaul.

Reply:
Added.